.github/workflows/build-and-test.yml

```diff
@@ -80,7 +80,7 @@ jobs:
         -DTTMLIR_ENABLE_RUNTIME=ON \
         -DTTMLIR_ENABLE_RUNTIME_TESTS=ON \
         -DTTMLIR_ENABLE_STABLEHLO=ON \
-        -DTTMLIR_ENABLE_OP_MODEL=ON \
+        -DTTMLIR_ENABLE_OPMODEL=ON \
```

Comment:
Is this change intentional?
TTMLIR_ENABLE_OP_MODEL -> TTMLIR_ENABLE_OPMODEL

Reply:
yes this is intentional, it was a bug. thanks for checking!